### DISCUSSION TITLE
Card/repository missing in HACS

### DISCUSSION THREAD

**Discussion by picassoruiz** [ulic75/power-flow-card#141] [Q&A] [answered]
Hi,
I'm searching for a dashboard that shows the energy flow and this one looks very promissing.

Unfortunately I cannot find it in HACS. What is the name of it? How can I add it?

Here is what I see when I search for it in HACS:
![image](https://user-images.githubusercontent.com/63224198/220133059-5c5cdc79-a968-4f2e-8b20-1c667b788e8a.png)

Thanks for support

**Comment by ulic75** [comment, marked as answer]
Looks like you're searching "Integrations" in HACS.  You need to search in "Frontend".

**Comment by picassoruiz** [comment]
You've made my day 🍻 
Now I found it and can play with it - sorry for such stupid question.
